### PR TITLE
ST-1677: Case Links Migration Config

### PR DESCRIPTION
**Pull request by sjamihmcts**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/ST-1677

### Change description ###
Case Links Migration Config

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by tomxelliott — CHANGES_REQUESTED** [review]
Ensure the clusters aren't running the migration job at the same time. 00 and 01 should probably alternate every 10 mins. Please also ensure this is the config for case flags as well

**Review by osheaf01 — APPROVED** [review]
LGTM

File: apps/sptribs/aat/00/kustomization.yaml [hunk 1]
```diff
@@ -3,6 +3,6 @@ kind: Kustomization
 namespace: sptribs
 resources:
 - ../base
-- ../../sptribs-cron-migrate-case-flags/sptribs-cron-migrate-case-flags.yaml
```
Comment (tomxelliott) [inline apps/sptribs/aat/00/kustomization.yaml:6]: keep the cron base files in here. See example in: apps/nfdiv/aat/00/kustomization.yaml

File: apps/sptribs/sptribs-cron-migrate-case-flags/aat/00.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ spec:
   releaseName: sptribs-cron-migrate-case-flags
   values:
     job:
-      schedule: "0/10 * * * *"
+      schedule: "15,45 * * * *"
```
Comment (tomxelliott) [inline apps/sptribs/sptribs-cron-migrate-case-flags/aat/00.yaml:9]: Aren't we going to restrict this job to run in the morning?
15,45 8-12 * * *

File: apps/sptribs/sptribs-cron-migrate-case-flags/aat/01.yaml [hunk 3]
```diff
@@ -6,7 +6,7 @@ spec:
   releaseName: sptribs-cron-migrate-case-flags
   values:
     job:
-      schedule: "0/10 * * * *"
+      schedule: "0,30 * * * *"
```
Comment (tomxelliott) [inline apps/sptribs/sptribs-cron-migrate-case-flags/aat/01.yaml:9]: Aren't we going to restrict this job to run in the morning?
15,45 8-12 * * *

File: apps/sptribs/sptribs-cron-migrate-case-links/aat/00.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: sptribs-cron-migrate-case-links
+spec:
+  releaseName: sptribs-cron-migrate-case-links
+  values:
+    job:
+      schedule: "20,50 * * * *"
```
Comment (tomxelliott) [inline apps/sptribs/sptribs-cron-migrate-case-links/aat/00.yaml:9]: Aren't you going to have this one in the afternoon?
20,50 12-16 * * *

File: apps/sptribs/sptribs-cron-migrate-case-links/aat/01.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: sptribs-cron-migrate-case-links
+spec:
+  releaseName: sptribs-cron-migrate-case-links
+  values:
+    job:
+      schedule: "25,55 * * * *"
```
Comment (tomxelliott) [inline apps/sptribs/sptribs-cron-migrate-case-links/aat/01.yaml:9]: Aren't you going to have this one in the afternoon?
25,55 12-16 * * *

File: apps/sptribs/aat/00/kustomization.yaml (hunk 1 above)
Comment (sjamihmcts) [reply apps/sptribs/aat/00/kustomization.yaml:6]: Initially we added here but still cron job is was not getting triggered. So we just moved to base file just to see if it creates the cron job in AAT.

**Review by tomxelliott — CHANGES_REQUESTED** [review]
Some changes needed

File: apps/sptribs/sptribs-cron-migrate-case-links/aat/01.yaml (hunk 5 above)
Comment (sjamihmcts) [reply apps/sptribs/sptribs-cron-migrate-case-links/aat/01.yaml:9]: ok Tom, will update this afternoon for now. Once flags migration is done for 1 case, we will update links.

File: apps/sptribs/sptribs-cron-migrate-case-flags/aat/00.yaml (hunk 2 above)
Comment (sjamihmcts) [reply apps/sptribs/sptribs-cron-migrate-case-flags/aat/00.yaml]: Updated to 8-16 for Case flags as the cron job is ready for Flags.

File: apps/sptribs/sptribs-cron-migrate-case-flags/aat/01.yaml (hunk 3 above)
Comment (sjamihmcts) [reply apps/sptribs/sptribs-cron-migrate-case-flags/aat/01.yaml]: Updated to 8-16 for Case flags as the cron job is ready for Flags.

File: apps/sptribs/sptribs-cron-migrate-case-links/aat/00.yaml (hunk 4 above)
Comment (sjamihmcts) [reply apps/sptribs/sptribs-cron-migrate-case-links/aat/00.yaml]: Updated to 16-18 for Case links as the case api task code is still in progress for Links.

File: apps/sptribs/sptribs-cron-migrate-case-links/aat/01.yaml (hunk 5 above)
Comment (sjamihmcts) [reply apps/sptribs/sptribs-cron-migrate-case-links/aat/01.yaml]: Updated to 16-18 for Case links as the case api task code is still in progress for Links.

**Review by tomxelliott — DISMISSED** [review]

**Review by willwatters — APPROVED** [review]

**Review by colmlynch60 — APPROVED** [review]
LGTM

**Review by osheaf01 — APPROVED** [review]
LGTM

**Review by cakeben — APPROVED** [review]

**Pull request merged**